.gitignore

```diff
@@ -17,6 +17,5 @@ MANIFEST
 vendor/*
 .DS_Store
 
-#binary - go build
-helm-service
```

Comment:
I would also add the binary file in .gitignore - e.g., when you build it locally (for whatever reason)

Reply:
I deleted double entries from the .gitignore file, the `helm-service` entry should be at the right location again